src/codegate/pipeline/codegate_context_retriever/codegate.py

```diff
@@ -131,11 +131,13 @@ async def process(  # noqa: C901
             messages = request.get_messages()
             filtered = itertools.dropwhile(lambda x: x[0] < last_user_idx, enumerate(messages))
             for i, message in filtered:
-                message_str = "".join([
-                    txt
-                    for content in message.get_content()
-                    for txt in content.get_text()
-                ])
+                message_str = ""
+                for content in message.get_content():
+                    txt = content.get_text()
+                    if not txt:
+                        logger.debug(f"content has no text: {content}")
+                        continue
```

Comment:
I found a traceback while asking Continue+Anthropic to evaluate dependencies in requirements.txt. There is also a unit test with exactly those contents.